spec/models/user_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject do
+    User.new(
+      name: 'Mateo Lane',
+      photo: 'https://unsplash.com/photos/mateopic',
+      bio: 'Comedian, Italian, stylish, and part time youtuber',
+      posts_counter: 5
+    )
+  end
+
+  before { subject.save }
+
+  it 'shows that name should be present ' do
+    subject.name = 'Mateo Lane'
+    expect(subject).to be_valid
+  end
+
+  it 'is not valid if name is not present' do
+    subject.name = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'shows that bio should be present ' do
+    subject.bio = 'Comedian, Italian, stylish, and part time youtuber'
+    expect(subject).to be_valid
+  end
+
+  it 'is not valid if bio is not present ' do
+    subject.bio = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'accepts the value if it is a string link' do
+    subject.photo = 'https://unsplash.com/photos/mateopic'
+    expect(subject).to be_valid
+  end
+
+  it 'is not valid if photo is not present' do
+    subject.photo = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'accepts the value if it is a integer' do
+    subject.posts_counter = 5
+    expect(subject).to be_valid
+  end
+
+  it 'is not valid if posts_counter is a string' do
+    subject.posts_counter = 'This is not a number'
+    expect(subject).to_not be_valid
+  end
```

Comment:
- [ ] Great job so far 👏🏻  I can see that the model specs have **Validation** However you should make sure that  Any custom methods in the User model have a corresponding unit test. If you check your User model you can see that you did not add a unit test for this method ⬇️ Let's add this.

```ruby
  def check_last_three_posts
    posts.limit(3).order(created_at: :desc)
  end
``